app/models/project.js

```diff
@@ -20,19 +22,46 @@ export const EmptyFeatureCollection = {
   }],
 };
 
+// (property,key) basic checks
 const hasAnswered = property => property === true || property === false;
 const hasFilledOut = property => !isEmpty(property);
+const hasFilledOutAndProposedDifferingZoning = function(property, key) {
+  debug(`
```

Comment:
should this debug be in the final pass?

Reply:
Not sure. They get stripped out of production builds. I can remove them if they're noisy (they probably are).

Reply:
Removed the debugs